pagerduty/resource_pagerduty_team_membership.go

```diff
@@ -30,6 +30,12 @@ func resourcePagerDutyTeamMembership() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 			},
+			"role": {
```

Comment:
Validation could probably be added:

> The role of the user on the team.
> Can be `observer`, `responder` or `manager`

https://api-reference.pagerduty.com/#!/Teams/put_teams_id_users_user_id